Documentation/installation/k8s-install-helm.rst

```diff
@@ -86,21 +86,20 @@ Install Cilium
        .. include:: ../installation/requirements-aks.rst
 
        **Install Cilium:**
-
        Deploy Cilium release via Helm:
```

Comment:
Unrelated, and undesirable. I didn't check the preview but I expect that removing this line will make Sphinx concatenate both line as a single paragraph, when the first one is meant to be some sort of sub-heading.